packages/core/package.json

```diff
@@ -22,8 +22,8 @@
     "type-check": "tsc --noEmit",
     "prettier-check": "prettier --check \"./**/*.ts*\"",
     "test": "pnpm test:node && pnpm test:edge && pnpm test:ui",
-    "test:edge": "vitest --config vitest.edge.config.js --run",
-    "test:node": "vitest --config vitest.node.config.js --run",
+    "test:edge": "vitest --config vitest.edge.config.js --run --threads=false",
+    "test:node": "vitest --config vitest.node.config.js --run --threads=false",
```

Comment:
Requried for langchain test stability. Otherwise node process hangs forever.

Reply:
Is this an `ai` / `vitest` issue, or something with langchain?